.github/workflows/nodejs.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       matrix:
         config:
-          - { os: macos-11, arch: x64 }
+          - { os: macos-13, arch: x64 }
           - { os: ubuntu-latest, arch: x64 }
-          - { os: windows-latest, arch: x64 }
-          - { os: windows-latest, arch: ia32 }
+          - { os: windows-2019, arch: x64 }
+          - { os: windows-2019, arch: ia32 }
```

Comment:
wasn't really sure what version to set this to be seemed like the [master branch](https://github.com/digidem/mapeo-desktop/blob/6769cd520e9907ac1b573074a6b8c1ed70bef89c/.github/workflows/nodejs.yml#L19-L20) updated this to 2019 as well, so went with that

Reply:
This change was probably to pin the version, since `windows-latest` became `windows-2022` and now `windows-2025`. Since this works, let's leave this as `windows-2019` for now, however note that this will likely be deprecated soon, since apparently [Github supports the latest 2 LTS versions](https://github.com/actions/runner-images/discussions/5343) and [Windows 2025 is now released](https://www.microsoft.com/en-us/windows-server/blog/2024/11/04/windows-server-2025-now-generally-available-with-advanced-security-improved-performance-and-cloud-agility/) (previous LTS were Windows 2022 and 2019).